apps/cli/src/cli/export.ts

```diff
@@ -54,7 +50,7 @@ function makeWordExportCLI(program: Command) {
     .argument('<article>', 'A link to the Curvenote article (e.g. OXA Link or API link)')
     .argument('[output]', 'The document filename to export to', '')
     .addOption(makeCleanOption())
-    .action(clirun(pathToWord, { program }));
+    .action(clirun(pathToWord, { program }, 3));
```

Comment:
What does the 3 do here?

Reply:
Ah, I see, slicing args. Thought it was on the other side of the bracket!

Reply:
Yeah, commander passes some giant object as the last argument. Cutting that off so it isn't passed to our functions...